pom.xml

```diff
@@ -4,12 +4,62 @@
   <groupId>org.goal</groupId>
   <artifactId>goal-service</artifactId>
   <version>1.0.0-SNAPSHOT</version>
+  <name>goal-service</name>
+  <url>http://maven.apache.org</url>
+  <parent>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-parent</artifactId>
+      <version>2.2.2.RELEASE</version>
+  </parent>
 <properties>
     <jdk.version>11</jdk.version>
     <maven.compiler.target>${jdk.version}</maven.compiler.target>
     <maven.compiler.source>${jdk.version}</maven.compiler.source>
 </properties>
   <dependencies>
-	
- </dependencies>
+      <!-- https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-starter-web -->
+      <dependency>
+          <groupId>org.springframework.boot</groupId>
+          <artifactId>spring-boot-starter-web</artifactId>
+          <version>2.2.4.RELEASE</version>
```

Comment:
Don't think we need to mention version here spring boot starter parent will bring appropriate compatible version.
Can you try removing the versions and try compiling

Reply:
Without version , springboot starter parent not working

Reply:
Done